src/material/core/tokens/_token-utils.scss

```diff
@@ -125,16 +125,19 @@ $_component-prefix: null;
 // TODO(crisbeto): should be able to replace the usages of `get-token-variable` with this.
```

Comment:
I guess this TODO should be resolved.

Reply:
Yeah this is probably as far as we'll get with switching usages over - the other cases legitimately try to get a var name